code/modules/mob/abstract/observer/observer.dm

```diff
@@ -926,4 +926,24 @@ mob/abstract/observer/MayRespawn(var/feedback = 0, var/respawn_type = null)
 		to_chat(usr, "<span class='danger'>The round hasn't started yet!</span>")
 		return
 
-	SSghostroles.vui_interact(src)
\ No newline at end of file
+	SSghostroles.vui_interact(src)
+
+/mob/abstract/observer/verb/submitpai() //Moves the ghost instead of just changing the ghosts's eye -Nodrak
+	set category = "Ghost"
+	set name = "Submit pAI personality"
+	set desc = "Submits pAI personality to pAI candidite pool"
+
+	if(!src.MayRespawn(TRUE))
```

Comment:
Maybe set a `MINISYNTH` or `ANIMAL` respawn type here? So people don't have to wait the full time to become a pAI.